nni/algorithms/compression/pytorch/quantization/quantizers.py

```diff
@@ -31,7 +31,7 @@ def validate_config(self, model, config_list):
 
         schema.validate(config_list)
 
-    def quantize_weight(self, wrapper, **kwargs):
+    def quantize_weight(self, input, wrapper, **kwargs):
```

Comment:
what is the meaning of `input`? and is it used in this function?...

Reply:
In this new version, input tensor's dynamic range will also be recorded to meet the requirement of tensorrt tensor range setting.

Reply:
so what is the meaning of `input`?

Reply:
`input` is input tensor of this op. It is used to calibrate the input tensor dynamic range. It won't be used in all quantizers so I have pass it by `kwargs`.